service-tests/nsp.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const Joi = require('joi')
+const ServiceTester = require('./runner/service-tester')
+
+const t = new ServiceTester({id: 'nsp', title: 'Node Security Platform'})
+
+t.create('get a package without vulnerabilities')
+  .get('/npm/bronze.json')
+  .expectJSONTypes(Joi.object().keys({name: 'nsp', value: 'no known vulnerabilities'}))
```

Comment:
Since these are both literals, you can write this a bit more concisely:

```js
.expectJSON({ name: 'nsp', value: 'no known vulnerabilities' })
```